src/main/c/ssl.c

```diff
@@ -1228,6 +1228,8 @@ static void ssl_info_callback(const SSL *ssl, int where, int ret) {
 TCN_IMPLEMENT_CALL(jlong /* SSL * */, SSL, newSSL)(TCN_STDARGS,
                                                    jlong ctx /* tcn_ssl_ctxt_t * */,
                                                    jboolean server) {
+    SSL *ssl = NULL;
```

Comment:
C89/ANSI requires declarations to be up front ... didn't' know we were compatible back to c89.